docs/jax.ops.rst

```diff
@@ -1,75 +1,15 @@
 
 jax.ops package
-=================
+===============
 
 .. currentmodule:: jax.ops
 
 .. automodule:: jax.ops
 
 .. _syntactic-sugar-for-ops:
 
-Indexed update operators
```

Comment:
Keep lines 11-52?

Reply:
They have no reason to be in this particular module once the deprecation is complete. Is the reason to point people to the new names?

Reply:
I added a paragraph that points to jnp.ndarray.at.